spark-commons/src/main/scala/za/co/absa/spark/commons/implicits/StructTypeImplicits.scala

```diff
@@ -417,15 +418,15 @@ object StructTypeImplicits {
     }
   }
 
-  implicit class StructTypeComparisonWhileIgnoringNullability(val schema: StructType) {
+  implicit class StructTypeComparisonWhileIgnoringNullability(val schema: StructType)(implicit errorColumnName: String = ErrorMessageArray.defaultErrorColumnName) {
```

Comment:
I'm not really sure about this implicit here - what if the val for this is named `errColumnName` or `errColName` or something like that? Why not treat it as an ordinary parameter (i.e. non implicit)?

Reply:
But, with that I was trying to at least get the value of the `errColName` from the ErrorMessageArray.

Reply:
I tried to do what you asking here, but unfortunately implicit classes takes only one parameter on their default constructor.
But in stead of parameterizing i just used `val errorColumnName = ErrorMessageArray.defaultErrorColumnName` to get the column name.